airflow/contrib/operators/bigquery_operator.py

```diff
@@ -75,4 +77,4 @@ def execute(self, context):
         conn = hook.get_conn()
         cursor = conn.cursor()
         cursor.run_query(self.bql, self.destination_dataset_table, self.write_disposition,
```

Comment:
PR looks great. Wondering if it would make sense to have explicit named parameters when calling the run_query (I'm not the python expert though).